Add info about order of touches in the array (#25822)
mdn/content: files/en-us/web/api/touchevent/touches/index.md

You can think of it as how many separate fingers are able to be identified as touching
the screen.

> One thing to note that the touches in `TouchEvent.touches[i]` doesn't neccessarily gives you i-th index of the array.Merely,the array is not ordered predictably and it depends upon implementation. A      developer cannot just **assume** a specific order,like the
> i-th element in the array being the i-th touch that happened,and the order in which touches
> happen is independent of the order of the list.To determine the latter,developers should rely
> on the the `touch` object IDs.
> 
> 
> 

Reviewer: ```suggestion
> **Note:** Touches inside the array are not necessarily ordered by order of occurrences (the
i-th element in the array being the i-th touch that happened). You cannot assume a specific order. To determine the order of occurrences of the touches, use the `touch` object IDs.
```

You can think of it as how many separate fingers are able to be identified as touching
the screen.

> **Note:** Touches inside the array are not necessarily ordered by order of occurrences (the

> i-th element in the array being the i-th touch that happened). You cannot assume a specific order. To determine the order of occurrences of the touches, use the `touch` object IDs.

Reviewer: ```suggestion
i-th element in the array being the i-th touch that happened). You cannot assume a specific order. To determine the order of occurrences of the touches, use the `touch` object IDs.

```
